src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -690,39 +663,36 @@ void GetInputLifeCycleCostNonrecurringCost(EnergyPlusData &state)
         // A1,  \field Name
         //      \required-field
         //      \type alpha
-        elcc->NonrecurringCost(iInObj).name = AlphaArray(1);
+        elcc->NonrecurringCost[iInObj].name = AlphaArray(1);
         // A2,  \field Category
         //      \type choice
         //      \key Construction
         //      \key Salvage
         //      \key OtherCapital
         //      \default Construction
-        if (UtilityRoutines::SameString(AlphaArray(2), "Construction")) {
-            elcc->NonrecurringCost(iInObj).category = costCatConstruction;
-        } else if (UtilityRoutines::SameString(AlphaArray(2), "Salvage")) {
-            elcc->NonrecurringCost(iInObj).category = costCatSalvage;
-        } else if (UtilityRoutines::SameString(AlphaArray(2), "OtherCapital")) {
-            elcc->NonrecurringCost(iInObj).category = costCatOtherCapital;
-        } else {
-            elcc->NonrecurringCost(iInObj).category = costCatConstruction;
+        elcc->NonrecurringCost[iInObj].category =
```

Comment:
All these changes seem fine really.  I definitely look forward to the BITF stuff becoming constexpr functions instead of macros.